### DISCUSSION TITLE
iPhone App: Audio doesn't route to external speaker

### DISCUSSION THREAD

**Discussion by midijohnny** [ggerganov/ggwave#31] [General]
[voca]: https://www.whathifi.com/pure/voca/review
I don't know if this possible to do - but it might be nice if so.
I connected my iPhone to my [voca] bluetooth speaker : but the Waver App doesn't direct the audio output to it.

(Other apps - like music/podcasts etc do)

**Comment by ggerganov** [comment]
Thanks for spotting this - I will look into fixing this.

**Reply by ggerganov** [reply]
I did some research and unfortunately this looks like something not very easy to achieve with the current implementation.
I am using libSDL to communicate with the the audio devices and I don't see a way to tell it to use a Bluetooth device or not.
Sorry about that.

**Reply by midijohnny** [reply]
No worries - thanks for looking at this and posting back. Cheers
